content/millennium/dstu2/individuals/patient.md

```diff
@@ -71,11 +75,28 @@ Notes:
 - The `birthdate` parameter may be provided twice to indicate a date range, but must contain the inclusive prefixes 'le' and 'ge'
 - The `birthdate` parameter may be provided once with the following prefixes: 'ge', 'le', 'gt', 'lt', 'eq'
 
-### Response
+### Headers
```

Comment:
This section will essentially be the same across all resources and actions except for cases where OAuth token isn't used like in conformance-related resources.  I'm guessing we're just trying to be explicitly clear rather than relying on the Overview.  Rather than copy-pasting the whole block, we could have another function call to get the text that essentially just swaps out "GET a Patient" for other resources/actions.